windows/src/developer/TIKE/actions/dmActionsMain.pas

```diff
@@ -935,19 +984,48 @@ procedure TmodActionsMain.actWindowPrevUpdate(Sender: TObject);
 procedure TmodActionsMain.DataModuleCreate(Sender: TObject);
 begin
   actFileOpen.Dialog.Filter :=
```

Comment:
Fixed up the filter for File Open when I saw it...

Reply:
Nice! This fixes the first point of #2127